src/data/mystery-encounters/mystery-encounter.ts

```diff
@@ -18,14 +18,16 @@ import {
   StatusEffectRequirement,
   WaveRangeRequirement
 } from "./mystery-encounter-requirements";
+import { BattlerIndex } from "#app/battle";
+import { EncounterAnim } from "#app/data/battle-anims";
 
 export enum MysteryEncounterVariant {
   DEFAULT,
   TRAINER_BATTLE,
   WILD_BATTLE,
   BOSS_BATTLE,
   NO_BATTLE,
-  SAFARI_BATTLE
+  REPEATED_ENCOUNTER
```

Comment:
not an optimal name.
How about `MULTIPLE_PHASES` for now?

Reply:
maybe `CONTINUOUS_ENCOUNTER` instead?